### ISSUE TITLE
A suggestion for moving through broken windows

### ISSUE THREAD

**Issue by Srcwya** [CleverRaven/Cataclysm-DDA#74843]
### Is your feature request related to a problem? Please describe.

So on the test i did i compared smashing the window and cleaning it ``(B,A,C)``
Vs
Running through a broken window
``(B,A,R)``
The results are..
``walking: 
BAR:582
BAC:462
running:
BAR:332
BAC:311``

The calculation was made looking at the move-cost counter on the mobile hud.
Basically for ``BAC`` it was (X+X+X+X)
And for ``BAR`` it was (X+X+X)
i was using the standard survivor starting gear, and holding nothing on my hands.
Getting cut in the window didnt seem to make a noticable change for the Move-Counter.

### Solution you would like.

If the player approaches a normal window and tries interacting with it an option to "jump through" the window be added. Instead of having to break the window (which would be three actions), the player will just immediately stand on the window tile and break the glass(reducing the move-cost by X% so that its faster than BAC), with a very high (maaaaaybe even 100%) chance of getting cut. Or have a chance to make the player trip and fall down (like a risky choice) without having to change the "get cut"%.

Basically a gamble, if you dare to you'll take X% less time to escape through the window but you could "snowball" it because you slipped or the injury caused you to go into "mild" pain

**Comment by kevingranade** [comment]
This is an incomprehensibly bad idea. This doesn't risk cutting yourself a little,  it would risk fatal cuts since you'd be dealing with large sheets of glass and your full body weight landing on them.
Jumping through a window is a movie fabrication.

**Comment by Srcwya** [comment]
> This is an incomprehensibly bad idea. This doesn't risk cutting yourself a little,  it would risk fatal cuts since you'd be dealing with large sheets of glass and your full body weight landing on them.
> Jumping through a window is a movie fabrication.

I mean yeah but if you are at the point that you're needing to break through windows as an escape tool a few 100 turns would count 
Also the way i meant wasnt that you *dive* through the window, but that you jump into the glass and then crawl into the building

Still, thanks for replying kevin!

If my suggestion doesnt seem good enough, at least please do something about the BAC and BAR issue
